src/launcher/java/org/truffleruby/launcher/Launcher.java

```diff
@@ -65,7 +65,7 @@
     public static final String LANGUAGE_ID = "ruby";
     public static final String ENGINE_ID = "truffleruby";
     public static final String LANGUAGE_VERSION = "2.3.5";
-    public static final String ENGINE_VERSION = System.getProperty("org.graalvm.version", BuildInformationImpl.INSTANCE.getRevision());
+    public static final String ENGINE_VERSION = System.getProperty("org.graalvm.version", "0.0-" + BuildInformationImpl.INSTANCE.getRevision());
```

Comment:
I can't say I like the "0.0", but if we don't know what the next GraalVM version is going to be, I don't have a better solution.